geodesy/package.xml

```diff
@@ -1,7 +1,8 @@
-<package>
-
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
   <name>geodesy</name>
-  <version>0.5.2</version>
+  <version>0.5.3</version>
```

Comment:
I agree with @tfoote's assessment that this package deserves a major bump for the ROS2 branch.

Reply:
Can do